docs/docs/building.md

```diff
@@ -27,89 +26,63 @@ contract MyContract is Migratable {
+This  will create a `zos.<network>.json` file with all the information specific to the chosen network. You can read more about this file in the [advanced topics](advanced.md#format-of-zosjson-and-zos-network-json-files) section.
 
-To create an upgradeable version of each of your contracts, run:
+To create an upgradeable version of our contract, we need to run:
 
-```
-zos create <contract_name_1> --network <network>
-zos create <contract_name_2> --network <network>
-...
-zos create <contract_name_n> --network <network>
+```sh
+zos create MyContract --init initialize --args 42 --network <network>
 ```
 
-This command takes an optional `--init` flag to call an initialization/migration
-function after you create the contract. In our simple example we want to pass a value (_e.g._: 42) to the initialize function:
+The `create` command takes an optional `--init` flag to call the initialization function after creating the contract, while the `--args` flag is to pass arguments to it. This way, we are initializing our contract with `42` as the value of the `x` state variable.
 
-```
-zos create MyContract --init initialize --args 42 --network development
-```
 
-After these simple steps, your upgradeable application is now on-chain!
+After these simple steps, our upgradeable application is now on-chain!
 
-## Upgrading your contracts
+## Upgrading our contract
 
-If, at a later stage, you want to upgrade your smart contracts' code in order to fix a bug or add a new feature, you can do it seamlessly using ZeppelinOS. 
+If, at a later stage, we want to upgrade our smart contract code in order to fix a bug or add a new feature, we can do it seamlessly using ZeppelinOS. 
 
-Note that while ZeppelinOS supports arbitrary changes in functionality, you will need to preserve all variables that appear in prior versions of your contracts, declaring any new variables below the already existing ones. You can find details on this in the [Advanced topics](advanced.md) page. 
+> **Note**: while ZeppelinOS supports arbitrary changes in functionality, you will need to preserve all variables that appear in prior versions of your contracts, declaring any new variables below the already existing ones. You can find details on this in the [advanced topics](advanced.md) page. 
 
-Once you've made the desired changes to your contracts, recompile them and push them to the network:
+Once we have made the desired changes to our contracts, we need to push them to the network:
 
-```
-npx truffle compile
+```sh
 zos push --network <network>
 ```
 
-Then, upgrade the already deployed contracts:
-
-```
-zos upgrade <contract_name_1> --network <network>
-zos upgrade <contract_name_2> --network <network>
-...
-zos upgrade <contract_name_n> --network <network>
-```
+> **Note**: bear in mind that the `push` command of the ZeppelinOS CLI also compiles our contracts. To avoid this, use the `--skip-compile` flag.
```

Comment:
sweet, but then shouldn't we change this also in the first appearance of: `npx truffle compile`?

I'd:
- keep the first `npx truffle compile`
- add a note *there* saying: bear in mind that the `push` command of the ZeppelinOS CLI already compiles the contracts, but we are compiling them explicitly here for clarity. If you want to prevent `zos push` from compiling the contracts, use the `--skip-compile` flag. 
- silently use `push` here, with no note.